server/data/localMockData/caseLoad.ts

```diff
@@ -0,0 +1,34 @@
+export type CaseLoad = {
```

Comment:
Do we want these types to live in `server/data/localMockData`? If we're building these to subsequently return the same shape from the real data I would expect to find these somewhere else otherwise these look like they'd only be the shape for mock data. We could put them potentially alongside the actual auth client or elsewhere and leave these files just for the data

Reply:
Good point. Maybe use `server/interfaces`?

Reply:
Good idea.